src/lib/SignatureHelper.ts

```diff
@@ -0,0 +1,110 @@
+import { soliditySha3 } from 'web3-utils';
+import { stripHexPrefix } from './BytesHelper';
+import { address } from '../../src/types';
+
+export const SIGNATURE_TYPES = {
```

Comment:
fyi you can use enums in typescript for this sort of thing. Those are more easily checked by the type checker so are preferred:

```typescript
export enum SIGNATURE_TYPES {
  INVALID = 0,
  ...
}
```